internal/controllers/s3userclaim/provisioner.go

```diff
@@ -131,6 +133,81 @@ func (r *Reconciler) ensureReadonlySubuser(ctx context.Context) (*ctrl.Result, e
 	return subreconciler.ContinueReconciling()
 }
 
+func (r *Reconciler) ensureOtherSubusers(ctx context.Context) (*ctrl.Result, error) {
+	specSubUsers := r.s3UserClaim.Spec.SubUsers
+	// Create a hashmap to move all spec and ceph subUsers to it
+	subUserFullIdSet := make(map[string]string)
```

Comment:
Here again, the `Set` suffix is an example of using Hungarian notation which is discouraged.

Reply:
Done